app/api/__init__.py

```diff
@@ -28,9 +29,10 @@
 # users
 api.route(UserList, 'user_list', '/users')
 api.route(UserDetail, 'user_detail', '/users/<int:id>', '/notifications/<int:notification_id>/user',
-          '/event-invoices/<int:event_invoice_id>/user')
+          '/event-invoices/<int:event_invoice_id>/user', '/speakers/<int:speaker_id>/user')
 api.route(UserRelationship, 'user_notification', '/users/<int:id>/relationships/notifications')
 api.route(UserRelationship, 'user_event_invoices', '/users/<int:id>/relationships/event-invoices')
+api.route(UserRelationship, 'user_speaker', '/users/<int:id>/relationships/speaker')
```

Comment:
One user can be related to multiple speaker profiles. This should be a list of speakers.